### PR TITLE
Change Event Name

### PR DESCRIPTION
**Pull request by sreynit02**
Issue: Some events' names don't make sense and don't correspond to the program name that they are associated with.

Fix: Change events names so they relate to their program. Also changed location to real buildings. 

Test: Run tests and all of them should pass.

### CODE REVIEW

File: database/test_data.py [hunk 1]
```diff
@@ -419,19 +419,19 @@
...
-        "location": "A Big Room",
+        "location": "Woodspen",
         "startDate": datetime.strptime("2021 6 12","%Y %m %d"),
         "endDate": datetime.strptime("2021 7 12","%Y %m %d")
     },
     {
         "id": 15,
         "term": 3,
-        "name": "Event 1",
+        "name": "Traing Event",
```
Comment (bryantal) [inline database/test_data.py]: *Training

File: database/test_data.py [hunk 2]
```diff
@@ -419,19 +419,19 @@
         "isTraining": True,
         "timeStart": datetime.strptime("6:00 pm", "%I:%M %p"),
         "timeEnd": datetime.strptime("9:00 pm", "%I:%M %p"),
-        "location": "A Big Room",
+        "location": "Woodspen",
```
Comment (bryantal) [inline database/test_data.py]: *Woods-Penniman

File: database/test_data.py [hunk 3]
```diff
@@ -310,55 +310,55 @@
...
         "term": 3,
-        "name": "Making Bowls",
-        "description": "Making Bowls Training",
+        "name": "Meet & Greet with Grandparent",
+        "description": "Students meet with grandparent for the first time",
         "isTraining": True,
         "timeStart": datetime.strptime("6:00 pm", "%I:%M %p"),
         "timeEnd": datetime.strptime("9:00 pm", "%I:%M %p"),
-        "location": "a big room",
+        "location": "Woodspen",
```
Comment (bryantal) [inline database/test_data.py]: *Woods-Penniman

**Review by bryantal — COMMENTED** [review]
Just some spelling errors

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
Thanks for making these changes! Alex found some spelling mistakes

**Review by bryantal — APPROVED** [review]

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**